components/proposal-list-distribution.tsx

```diff
@@ -0,0 +1,103 @@
+import { useEffect, useState } from "react";
+
+import BigNumber from "bignumber.js";
+import { useTranslation } from "next-i18next";
+
+import BountyDistributionItem from "components/bounty-distribution-item";
+
+import { truncateAddress } from "helpers/truncate-address";
+
+import { DistributedAmounts } from "interfaces/proposal";
+import { TokenInfo } from "interfaces/token";
+
+import { getCoinInfoByContract } from "services/coingecko";
+
+import { useAppState } from "../contexts/app-state";
+interface Props {
+  distributedAmounts: DistributedAmounts;
+}
+
+export default function ProposalListDistribution({
+  distributedAmounts,
+}: Props) {
+  const { t } = useTranslation(["common", "proposal"]);
+
+  const [coinInfo, setCoinInfo] = useState<TokenInfo>();
+
+  const { state } = useAppState();
+
+  async function getCoinInfo() {
+    await getCoinInfoByContract(state.Service?.network?.networkToken?.symbol)
```

Comment:
what you want is `getCoinPrice(state.Service?.network?.networkToken?.symbol)`

Reply:
its used on `handleConversion`, to convert the value by coingecko